### PR TITLE
removed io.realm.internal.Version class

### PR DESCRIPTION
**Pull request by zaki50**
`io.realm.internal.Version` is an relic of old era.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 * Added `RealmQuery.in()` for a comparison against multiple values.
 
+### Internal
+
+* Removed `io.realm.internal.Version` class.
```
Comment (cmelchior) [inline CHANGELOG.md]: I wouldn't mention this in the changelog. It doesn't impact users in any way.

**Comment by cmelchior** [comment]
Apart from comment 👍

File: CHANGELOG.md (hunk 1 above)
Comment (zaki50) [inline CHANGELOG.md]: removed

**Comment by kneth** [comment]
:+1:

**Pull request merged**